sound/soc/sof/loader.c

```diff
@@ -310,6 +310,8 @@ static void sof_get_windows(struct snd_sof_dev *sdev)
 	u32 outbox_size = 0;
```

Comment:
Commit subject prefix should be 'ASoC: SOF: loader' we always use SOF instead of Sof. Also please remove dot at the end of commit subject.
We should also explain better why do we need this.